test/spec/provider/camunda/InputOutputParameterTextSpec.js

```diff
@@ -727,6 +816,10 @@ function getCursorPosition() {
   return range.startOffset;
 }
 
+function getParameterScriptValueInput(container, id) {
+  return domQuery('input[id="input-parameter-' + id + 'cam-script-resource-val"]', getInputParametersGroup(container));
```

Comment:
So it's basically not the "script value", but the "script resource value"? Maybe we should make it clear then, also [in the test itself](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/401/files#diff-3f0f09702496b6cf60a8e76b7a70db2b75efd7aa92c265f073e70a944e4c84d9R700), to not get confused by the other "script value", namely the Inline Script.

`getParameterScriptResourceValue`

Reply:
True, will do